themes/janeswalk/js/components/pages/Dashboard/Walk.jsx

```diff
@@ -0,0 +1,36 @@
+//TODO: Fix dateFormatted sort
+import { dateFormatted } from './../itinerary/ItineraryUtils';
+
+//TODO: Common component from <Itinerary/> <Walk/> (Post PR)
+
+const Walk = ({title, start, meeting, id, team, url}) => {
+  //TODO: no consistent definition of "walk-leader", how to grab the 'Led by' data, grabbing first team member (Post PR)
+  //TODO: Publish | Edit | Unpublish (PR)
+  return (
+    <li>
+      <div className="walk">
+        <h3>{title}</h3>
+        <h4>{dateFormatted(start)}</h4>
+        <h4>Led by {`${team[0]['name-first']} ${team[0]['name-last']} ${team[0]['email']}`}</h4>
```

Comment:
Don't ever assume there will always be a first member in a team. We could potentially have walks with empty teams. This file's a good resource: https://github.com/jkoudys/janeswalk-web/blob/react14/models/page_types/Walk.php